frontend/web/components/modals/ConfirmRemoveWebhook.tsx

```diff
@@ -45,7 +45,8 @@ const ConfirmRemoveWebhook: FC<ConfirmRemoveWebhookType> = ({
                 }
               </strong>
               . You should ensure that you do not contain any references to this
-              webhook in your applications before proceeding.
+              webhook in your applications before proceeding. This action cannot
```

Comment:
Yeah, I think all of that text is pretty much unnecessary. Perhaps something like 

```
    This will remove webhooks to {url} for the environment{env}. This action cannot be undone.
```